### PR TITLE
add lazy g2g button

### LINKED ISSUE #11188: Button for admins to instantly offer a player to ghosts from adminhelps and prays.
To speed up the proccess by aprox 1 second, probably.

### PR DESCRIPTION
**Pull request by ghost**
closes #11188

### CODE REVIEW

**Review by yoyobatty — DISMISSED** [review]
Nice

File: hippiestation/code/modules/admin/topic.dm [hunk 1]
```diff
@@ -211,4 +216,4 @@
 			log_admin("[key_name(usr)] temporarily removed [ckey] as a donator, as the database was not connected.")
 	if(C && removedonator)
 		C.is_donator = FALSE
-		to_chat(C, "<span class='danger'>You have been removed as a donator!")
\ No newline at end of file
+		to_chat(C, "<span class='danger'>You have been removed as a donator!")
```
Comment (Jujumatic) [inline hippiestation/code/modules/admin/topic.dm]: !!!!

**Comment by Jujumatic** [comment]
nice code but it might be a cool idea if you could allow players to do this as well, as sometimes you g2g when there's no badmins around

**Comment by Jujumatic** [comment]
possibly just add it as an IC verb, next to "notes" or something like that?

**Comment by ghost** [comment]
yeah let the player "auction"  his body to ghosts at will,  no admin needed

File: hippiestation/code/modules/admin/topic.dm (hunk 1 above)
Comment (ghost) [reply hippiestation/code/modules/admin/topic.dm]: Newline

**Comment by ghost** [comment]
ok done, IC verb added

**Comment by JessHorserage** [comment]
THANKS BRAH!!

**Comment by Maroonenahmoron** [comment]
I don't see a lot of room for people to abuse this, or maybe I'm wrong? +1 for its intention.

**Comment by DaydreamPuppeh** [comment]
+1

**Comment by Manly-Dwarf** [comment]
+1, thanks for adding a cooldown and catban protection.

**Comment by cris5311** [comment]
+1

**Comment by mikolaj007** [comment]
+1

**Review by yoyobatty — APPROVED** [review]

**Pull request merged**